lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -330,7 +346,7 @@ Function Choco-Install
                 -ignorechecksums $ignorechecksums -ignoredependencies $ignoredependencies `
                 -allowdowngrade $allowdowngrade -proxy_url $proxy_url `
                 -proxy_username $proxy_username -proxy_password $proxy_password `
-                -allowprerelease $allowprerelease
+                -allowprerelease $allowprerelease  -username $username -password $password
```

Comment:
I would prefer the parameters are put together with `-source` (so reorder this a bit so these 3 parameters are together).

Reply:
(Same below)